pennylane/transforms/dynamic_one_shot.py

```diff
@@ -341,14 +349,13 @@ def gather_mcm(measurement, samples, is_valid):
             counts = qml.math.array(counts, like=interface)
             return counts / qml.math.sum(counts)
         if isinstance(measurement, CountsMP):
-            mcm_samples = [{"".join(str(v) for v in tuple(s)): 1} for s in mcm_samples]
+            mcm_samples = [{"".join(str(int(v)) for v in tuple(s)): 1} for s in mcm_samples]
         return gather_non_mcm(measurement, mcm_samples, is_valid)
+    mcm_samples = qml.math.ravel(qml.math.array(mv.concretize(samples), like=interface))
```

Comment:
>So the additional leading axis of size 1 in the non-ProbabilityMP case is not required, because it is ravelled away anyways, right?

Yes exactly.

> Was this previously an alias for at_least_1d?

I guess implicitly, yes, but I hadn't really thought about it that way 😅

Reply:
Thanks! :)